Source/WebCore/page/csp/CSPViolationReportBody.cpp

```diff
@@ -30,6 +30,7 @@
 #include "FormData.h"
 #include "SecurityPolicyViolationEvent.h"
 #include <wtf/IsoMallocInlines.h>
+#include <wtf/JSONValues.h>
```

Comment:
maybe stray?

Reply:
Intentional change to make sure the patch builds. Because of the way WebKit is built, whenever you add new files to the project, you end up finding missing includes and causing build failures. This is a bit annoying but supposedly it makes the build faster.